astroid/nodes/scoped_nodes/scoped_nodes.py

```diff
@@ -1376,6 +1378,7 @@ def argnames(self):
             names = _rec_get_names(self.args.arguments)
         else:
             names = []
+        names += [elt.name for elt in self.args.kwonlyargs]
```

Comment:
Didn't saw it earlier. Wouldn't it make sense to add them **after** `vararg`? That would keep the correct order.